drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1115,6 +1115,7 @@ function DragAndDropBlock(runtime, element, configuration) {
 
             if (attemptsRemain()) {
                 data.misplaced_items.forEach(function(misplaced_item_id) {
+                    state.last_action_correct = false
```

Comment:
I'll do some additional debugging to see if with #96 in place, `data.correct` is giving us what we need.